forge-gui/res/cardsfolder/upcoming/bladehold_cleaver.txt

```diff
@@ -0,0 +1,10 @@
+Name:Bladehold Cleaver
+ManaCost:2 R W
+Types:Artifact Equipment
+K:For Mirrodin
+S:Mode$ Continuous | Affected$ Creature.EquippedBy | AddPower$ 2 | AddToughness$ 2 | Description$ Equipped creature gets +2/+2.
+T:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Graveyard | ValidCard$ Creature.EquippedBy | Execute$ TrigDraft | TriggerDescription$ Whenever equipped creature dies, draft a card from CARDNAME's spellbook.
+SVar:TrigDraft:DB$ Draft | Spellbook$ Accorder Paladin,Ardent Recruit,Auriok Sunchaser,Blade-Tribe Berserkers,Goblin Gaveleer,Hero of Bladehold,Hero of Oxid Ridge,Mirran Crusader,Oxidda Scrapmelter,Sunspear Shikari,Oxidda Finisher,Barbed Batterfist,Bladehold War-Whip,Dragonwing Glider,Jor Kadeen; the Prevailer
+DeckHas:Ability$Token & Type$Rebel|Cat|Soldier|Beast|Knight|Goblin|Beserker
+K:Equip:3
+Oracle:For Mirrodin\n Equipped creature gets +2/+2.\n Whenever equipped creature dies, draft a card from Bladehold Cleaver's spellbook.\nEquip {3}.
```

Comment:
Isn't it `For Mirrodin!` (with !)

I don't think you want a space after `\n` newline